packages/core/src/index.ts

```diff
@@ -1,15 +1,12 @@
-export { SimulatedClock } from './SimulatedClock.ts';
-export { isMachineSnapshot, type MachineSnapshot } from './State.ts';
-export { StateMachine } from './StateMachine.ts';
-export { StateNode } from './StateNode.ts';
 export * from './actions.ts';
 export * from './actors/index.ts';
 export { assertEvent } from './assert.ts';
 export {
   Actor,
   createActor,
   interpret,
-  type Interpreter
+  type Interpreter,
+  type RequiredOptions
```

Comment:
this type should get renamed to be more specific if we want to export it

Reply:
```ts
export type RequiredOptions<TLogic extends AnyActorLogic> =
  undefined extends InputFrom<TLogic> ? never : 'input';
```

Something like `ActorHasRequiredInputOption`?

Reply:
This one returns a union of required options - not a boolean. So I'd go with `RequiredActorInstantiationOptions` or smth like this.

Reply:
Opted for `RequiredActorInstanceOptions`